### PR TITLE
#164955276 create models and seed database

### PR DESCRIPTION
**Pull request by vic3king**
#### What does this PR do?
creates Database models and seeders

#### Description of Task to be completed?
- Create database models
- Create associations between each model
- Seed database
- Fix typo in sequelizerc

#### How should this be manually tested?
Clone the Repo
CD into ft-generate-models-164955276
NPM run migrate: undo
NPM run migrate
NPM run seed

#### What are the relevant pivotal tracker stories?
[164955276](https://www.pivotaltracker.com/n/projects/2318856/stories/164955276)

### CODE REVIEW

File: server/migrations/20190331064122-create-user.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        unique: true,
+      },
+      bio: {
+        type: Sequelize.TEXT,
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      isactivated: {
```
Comment (rotimi-babalola) [inline server/migrations/20190331064122-create-user.js]: Change this to `is_activated` - use snake case

File: server/migrations/20190331141313-create-reported-article.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Reporteds', {
```
Comment (rotimi-babalola) [inline server/migrations/20190331141313-create-reported-article.js]: Reporteds???

File: server/migrations/20190331203102-create-draft-article.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (rotimi-babalola) [reply server/migrations/20190331203102-create-draft-article.js]: This table is a duplicate of the `Articles` table. You can add an `isDraft` column to the the `Articles` table to determine if an article is a draft or not.

**Review by rotimi-babalola — COMMENTED** [review]
Overall this is good. But just make the required changes

**Comment by rotimi-babalola** [comment]
Update Users table to track user's followers

**Comment by vic3king** [comment]
> Overall this is good. But just make the required changes

Thank You. Changes have been Implemented.

**Pull request merged**